app/views/carts/show.html.erb

```diff
@@ -43,9 +43,22 @@
       </tbody>
     </table>
 
-    <div class="user-links">
-      <!--  checkout button goes here -->
+
+<% if current_user %>
+  <h5 id="user-message">Checkout</h5>
+  <div class="user-checkout">
+    <%= button_to "Check Out", orders_path, method: :post %>
+  </div>
+
+<% else %>
+  <h5 id='guest-message'>Login or Create Account to Checkout</h5>
+  <div class="row guest-links">
+    <div class="col s1">
+      <%= button_to "Login", login_path, method: :get, class: "waves-effect waves-light btn" %>
     </div>
+    <div class="col s1 offset-s1">
+      <%= button_to "Create Account", new_user_path, method: :get, class: "waves-effect waves-light btn" %>
+
 
     <% unless current_user %>
```

Comment:
We could probably take this second conditional out because it's checking for the same thing that the if current_user conditional checks for. Good thing to do in refactoring, not necessary now.